### DISCUSSION TITLE
Add "Add Parent node" as an option to context menu of nodes in a scene

### DISCUSSION THREAD

**Discussion by Dancovich** [godotengine/godot-proposals#8268] [Editor]
Add an option "Add Parent node" to context menus of nodes in a scene.

This option would basically be a shortcut to adding a sibling node to the node we selected and then re-parenting this selected node to the new one.

![image](https://github.com/godotengine/godot-proposals/assets/1204296/5d2f1a7f-479e-4d63-be65-940d9406f60e)

This would be specially useful when developing user interfaces. It's very common that we layout the basic structure of an user interface using Control nodes, but then we start customizing stuff, like adding a MarginContainer around an existing element or wrapping an existing control in a VBox or HBox when we decide to add other controls in that same position.

**Comment by jmarceno** [comment]
Doesn't "Reparent to New Node"  already do that?

![image](https://github.com/godotengine/godot-proposals/assets/4137615/d6835d34-785a-4253-8894-29605262430c)

**Reply by AThousandShips** [reply]
There is indeed exactly this feature available, and with a IMO more descriptive name, "add parent node" implies that you add *another* parent, rather than inserting a parent

**Reply by Dancovich** [reply]
I'm sorry, I completely misinterpreted what this option was supposed to do. This proposal can be closed.

A little nitpick, the reason I missed that option is because it lacks the ellipsis character at the end (...). The editor default is that options that don't act right away and instead open a dialog will have an ellipsis, but this option doesn't.

![image](https://github.com/godotengine/godot-proposals/assets/1204296/eefdc466-4d90-4251-819b-51e3b6c02a8f)

**Comment by Dancovich** [comment]
Closed because the option already exists

**Comment by mjasnikovs** [comment]
Please open it, there are no such options for "top" node.

![image](https://github.com/godotengine/godot-proposals/assets/12757881/43e53061-b48a-488b-a680-fd35f31cbcaf)
